onebusaway-android/build.gradle

```diff
@@ -50,8 +56,12 @@ android {
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 21
-        versionCode 76
-        versionName "2.1.9"
+        versionCode 84
+        versionName "2.2.3"
+    }
+
+    dexOptions {
```

Comment:
@acrown-msft I think we can get rid of `dexOptions` now that multidex is removed?